sympy/integrals/rationaltools.py

```diff
@@ -22,7 +28,7 @@ def ratint(f, x, **flags):
     References
     ==========
 
-    .. [Bro05] M. Bronstein, Symbolic Integration I: Transcendental
+    .. [1] [Bro05] M. Bronstein, Symbolic Integration I: Transcendental
```

Comment:
If it already have named explicit citation name under brackets `[Bro05]`, it should not be enumerated twice.
`[1]` can possibly ignore `[Bro05]` than doubly linking

Reply:
Done.